src/PresentationalComponents/Export/SystemsPdfBuild.js

```diff
@@ -102,6 +103,7 @@ export const tablePage = ({ page, systems, intl }) => {
   ];
 
   return (
+    // that fragment is not needed at all/
```

Comment:
When working on these I found this feature isnt working. The network request fails on select, need to look into further